user-guide/Standard_Apps/IDP/Tutorials/IDP_Tutorial_TakeConfigurationBackup.md

```diff
@@ -0,0 +1,200 @@
+
+  - The latest version of IDP is installed.
+
+    > [!TIP]
+    > See [Installing DataMiner IDP](xref:Installing_DataMiner_IDP)
+
+- A shared folder must be accessible from the system to store the backups.
+
+> [!IMPORTANT]
+> Do not use a DaaS system to follow this tutorial. On DaaS systems, it is not possible to configure a shared folder or access a shared folder outside the DaaS system.
+
+## Overview
+
+- [Step 1: Create the CI Type](#step-1-create-the-ci-type)
+- [Step 2: Create an element and manage it](#step-2-create-an-element-and-manage-it)
+- [Step 3: Set up the configuration archive](#step-3-set-up-the-configuration-archive)
+- [Step 4: Create the script to retrieve the configuration backup](#step-4-create-the-script-to-retrieve-the-configuration-backup)
+- [Step 5: Retrieve the device configuration](#step-5-retrieve-the-device-configuration)
+
+## Step 1: Create the CI Type
+
+First, a [CI Type](xref:CI_Types) needs to be created for IDP to know how to manage the element or device.
+
+Instead of creating the CI Type from scratch, you will generate it based on an existing connector. In this example, the [Generic Switch](https://catalog.dataminer.services/details/connector/931) connector is used, but you can use any other connector.
+
+1. Deploy the connector to start from:
+
+   1. Deploy the latest version of the [Generic Switch](https://catalog.dataminer.services/details/connector/931) connector from the DataMiner Catalog.
+
+      > [!TIP]
+      > See also: [Deploying a Catalog item to your system](xref:Deploying_a_catalog_item)
+
+   1. Make sure the latest version is [set as production](xref:Promoting_a_protocol_version_to_production_version).
+
+      ![Generic Switch connector](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_ProtocolTemplates.png)
+
+1. Generate the CI Type:
+
+   1. Open the IDP app.
+
+   1. Go to *Admin* > *CI Types*.
+
+   1. Above the table on the right, click *Generate* to start the wizard.
+
+      ![Start to generate the CI Type script](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_GenerateCiType_0.png)
+
+   1. In the wizard, select the connector and click *Generate*.
+
+      ![Select the connector](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_GenerateCiType_1.png)
+
+   1. Click *Finish* to close the wizard.
+
+   1. Switch off the *Show All* button in the top-right corner to locate the generated CI Type more easily.
+
+      ![Generate the CI Type](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_GenerateCiType_2.png)
+
+## Step 2: Create an element and manage it
+
+1. [Create an element](xref:Adding_elements) that uses the connector you deployed in the previous step.
+
+1. In the IDP app, go to *Inventory* > *Unmanaged*.
+
+1. Click *Refresh* to refresh the table.
+
+   > [!NOTE]
+   > Make sure *Detected CI Type* contains the generated CI Type. If it does not, confirm that the element was created with the production version.
```

Comment:
I tested this on my own DMA, and the Detected CI Type column did not contain the generated CI Type even though the element was created with the production version. Any idea what could be the cause of this?

Reply:
For this to happen the connector version in the CI Type does not match the connector version used to create the element. Or, there was an exception.
Please check if the versions match. Or, you have no exceptions in the log file of the *DataMiner IDP* element.

Reply:
After some investigation, this issue was discovered to be due to a casing mismatch; the protocol on my DMA was "Generic switch" instead of "Generic Switch".